packages/legend-application-studio/style/components/editor/uml-editor/_uml-element-editor.scss

```diff
@@ -180,6 +217,28 @@
     }
   }
 
+  &__drag-handler {
+    @include flexCenter;
+
+    flex: 0;
+    height: 2.8rem;
+    color: var(--color-dark-grey-500);
+    border-radius: 0.2rem;
+    margin-left: 0;
+    padding-top: 0.1rem;
+    padding-left: 0.5rem;
+    margin-right: 0.5rem;
+    cursor: pointer;
+  }
+
+  &__drag-handler:hover {
+    color: var(--color-dark-grey-400);
+  }
+
+  &__drag-handler:active {
```

Comment:
since this is just a `div`, I don't think we need this

Reply:
fair 👍